plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -37,7 +37,9 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
-        return BigQueryQueryRunner.createQueryRunner(ImmutableMap.of());
+        return BigQueryQueryRunner.createQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.of("bigquery.case-insensitive-name-matching", "true"));
```

Comment:
i suggest dedicated class for this, especially since this is not enabled by default